tests/test_cloudformation/test_cloudformation_stack_crud_boto3.py

```diff
@@ -2462,6 +2462,65 @@ def test_create_and_update_stack_with_unknown_resource():
             )
 
 
+@mock_aws
+def test_valid_change_set_name():
```

Comment:
I don't think this test is necessary? There are other tests that create a changeset, so they verify implicitly that the name is valid